.eslintrc.json

```
@@ -0,0 +1,32 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:prettier/recommended",
+      "airbnb",
+      "airbnb/hooks"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2020,
+      "sourceType": "module",
+      "ecmaFeatures": {
+        "jsx": true
+      }
+    },
+    "rules": {
+      "prettier/prettier": ["error", { "singleQuote": false }],
+      "react/prop-types": "off",
+      "@typescript-eslint/explicit-module-boundary-types": "off",
+      "@typescript-eslint/no-explicit-any": "off",
+      "quotes": ["error", "double", { "avoidEscape": true, "allowTemplateLiterals": true }],
+      "jsx-quotes": ["error", "prefer-double"]
```

> eslint와 prettier를 처음 접해 아직 잘 다루지 못하는 것 같습니다. eslint에서는 airbnb룰을 지정하나, string형에서는 쌍따옴표를 허용하도록 지정해보았는데 맞게 작성한 것인지 궁금합니다.

PR 내용에 적어주신 질문에 대한 답변을 여기에 달아둘게요. 각 도구에 대한 구별은 지금 시작하는 단계에서는 아래처럼 이해하고 계셔도 충분합니다.

- eslint : 코드를 정적 분석해서 문제가 되는 부분을 알려주는 도구 (e.g. 사용하지 않는 변수에 대한 주의 표시)
- prettier : 코드의 포맷팅을 일관되도록 정리해주는 도구 (e.g. 코드의 줄바꿈, 세미 콜론 사용 여부 등등)

여기서 eslint에도 일부분 prettier가 가진 포맷팅을 처리해줄 수 있기 때문에 둘 간의 충돌 방지를 위해 eslint-plugin-prettier을 적용해서 eslint가 가진 포맷팅 관련 rule을 비활성화 하게 됩니다. ([eslint-config-prettier 코드 참고](https://github.com/prettier/eslint-config-prettier/blob/main/index.js#L23-L214)) 포맷팅은 prettier에서만 처리하도록 하는거죠. prettier에서는 single quote를 사용하도록 설정했는데 활용하는 eslint config에서는 double quote를 사용하도록 하게 되면 충돌이 날테니까요. 

지금은 다른 코멘트에서 피드백 드린 내용을 반영하고 19번 라인의 `prettier/prettier`와 23번 라인의 `quotes` 그리고 24번 라인의 jsx-quotes를 아래처럼 제거하시는게 좋겠습니다.

```suggestion
      "react/prop-types": "off",
      "@typescript-eslint/explicit-module-boundary-types": "off",
      "@typescript-eslint/no-explicit-any": "off",
```